test/renaming/test.sh

```diff
@@ -0,0 +1,65 @@
+#/bin/bash
+#
+# Script to test whether the StackState egnt contains no visible references to DataDog
+# (only to StackState)
+#
+# Arguments:
+#   1) package name
+#
+
+set -e
+
+# parameter check
+if [ "$#" -ne 1 ];  then
+  echo "Parameter error."
+  echo "Usage: ./test/sh <package>"
+  echo "Example: ./test.sh pkg/stackstate-agent_1.0.0.git.3.d95a036-1_amd64.deb"
+  exit 1
+fi
+
+# set variables based on parameters
+ARTIFACT=$1
+PROJECT_DIR=`pwd`
+
+# artifact present?
+echo "Checking artifact's existence."
+if [ ! -e ${ARTIFACT} ]; then
+  echo "Artifact not found at ${ARTIFACT}."
+  exit 2
+fi
+
+cp ${ARTIFACT} agent.deb
+
+rm -rf unpacked
+mkdir unpacked
+cd unpacked
+
+ar xv ../agent.deb
+tar xvf control.tar.gz || true
+tar xvf data.tar.gz || true
+
+# Stuff in embedded is not important
+EMBEDDED_DIR="opt/stackstate-agent/embedded/lib/"
+
+find . -iname \*datadog\* \
+  | grep -v "$EMBEDDED_DIR" \
+  | grep -v "/opt/stackstate-agent/bin/agent/dist/views/private/images/datadog_icon_white.svg" \
+  | grep -v "/opt/stackstate-agent/LICENSES/go_dep-gopkg.in_zorkian_go-datadog-api" \
+  | grep -v "/opt/stackstate-agent/LICENSES/go_dep-github.com_DataDog_agent-payload" \
+  | grep -v "/opt/stackstate-agent/LICENSES/go_dep-github.com_DataDog_gohai" \
+  | grep -v "/opt/stackstate-agent/LICENSES/go_dep-github.com_DataDog_zstd" \
+  | tee -a out.txt
+find . -iname \*dd-\* | tee -a out.txt
+
+echo "Output:"
+cat out.txt
+echo "end"
+
+# Verify we found no references to datadog we did not expect
+if [ -s out.txt ]
+then
+  echo "Test failed"
```

Comment:
more descriptive log: `echo "Please fix branding: there is still something using (dd- | datadog) prefix"`

Reply:
done